keyboards/nullbitsco/nibble/keymaps/snailmap/keymap.c

```diff
@@ -0,0 +1,552 @@
+        _______,   _______,    _______, _______, _______, _______, _______, KC_PPLS, KC_4,    KC_5,    KC_6,       _______,       _______,          _______, _______,
+        _______,   _______,    _______, _______, _______, _______, _______, KC_PMNS, KC_1,    KC_2,    KC_3,       _______,       _______,          _______, _______,
+        _______,   _______,    _______, _______,                   KC_0,                      KC_PSLS, _______,    _______,       _______,          _______, _______
+    ),
+    [_L3] = LAYOUT_ansi(
+                   _______,    KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,      KC_F10,        KC_F11,  KC_F12,  _______, _______,
+        RGB_TOG,   _______,    _______, _______, _______, _______, _______, _______, _______, _______, _______,    _______,       _______, _______, _______, _______,
+        _______,   _______,    _______, _______, _______, _______, _______, _______, _______, _______, _______,    _______,       _______,          _______, _______,
+        KC_SYSREQ, _______,    _______, _______, _______, _______, _______, _______, _______, _______, _______,    _______,       _______,          _______, _______,
+        _______,   _______,    _______, _______,                   _______,                   _______, _______,    _______,       _______,          _______, _______
+    )
+};
+
+#ifdef OLED_DRIVER_ENABLE
+/*===========================================    OLED CONFIGURATION    ===========================================*/
+bool  oled_horizontal   = true;         // OLED rotation  (true = horizontal,  false = vertical)
+bool  ansi_layout       = true;         // ANSI or ISO layout  (true = ANSI,  false = ISO)
+bool  split_space       = true;         // Split spacebar  (true = split spacebar,  false = 6.25u or 7u spacebar)
+bool  three_mods_left   = true;         // Left mods layout  (true = 3x 1.25u keys,  false = 2x 1.5u keys)
+bool  three_mods_right  = false;        // Right mods layout  (true = 3x 1u keys,  false = 2x 1.5u keys)
+bool  graph_direction   = true;         // Graph movement  (true = right to left,  false = left to right)
+float graph_top_wpm     = 100.0;        // Minimum WPM required to reach the top of the graph
+int   graph_refresh     = 1000;         // In milliseconds, determines the graph-line frequency
+int   icon_med_wpm      = 50;           // WPM required to display the medium snail
+int   icon_fast_wpm     = 72;           // WPM required to display the fast snail
+char  MA_name[7]        = "QWERTY";     // Layer _MA name (6 letters max, gets shortened to 5 if vertical layout)
+char  L1_name[7]        = "ARROWS";     // Layer _L1 name (6 letters max, gets shortened to 5 if vertical layout)
+char  L2_name[7]        = "NUMPAD";     // Layer _L2 name (6 letters max, gets shortened to 5 if vertical layout)
+char  L3_name[7]        = "FUNCTN";     // Layer _L3 name (6 letters max, gets shortened to 5 if vertical layout)
```

Comment:
You should make these defines, rather than chars

Reply:
Changed